docs/installing-northstar/northstar-installers/viper-guide.md

```diff
@@ -0,0 +1,71 @@
+# Viper Guide
+
+Viper is a mod manager/installer for Northstar, a community made mod for Titanfall 2, that automatically updates mods and Northstar versions.
+
+Similar to FlightCore, Viper works on both Windows and Linux and has similar use cases, mostly differing in downloading Viper.
+
+## Windows Installation
+
+![The Windows download button on Viper's GitHub page](../../images/viper-windows-download.png)
+
+Install Viper from the [Northstar website](https://northstar.tf) or the Windows download button on [Viper's main GitHub page](https://github.com/0neGal/viper).
+
+After this, run the file you downloaded to install Viper, then, if it doesn't automatically, open Viper.
+
+## Linux Installation
+
+To install Viper on Linux, you can either install the flatpak, or the `AppImage`. The flatpak version can be installed simply by using the command `flatpak install viper` in your terminal, which you can then open with your system. Viper's Flathub page can be found [here](https://flathub.org/apps/com.github._0negal.Viper).
+
+To use the `AppImage` version, you want to download the newest `AppImage` file from the [Viper release page](https://github.com/0neGal/viper/releases), store it in a place that you want to, and make sure to right click on it, click on properties and tick `Executable as Program` on the AppImage file as shown below to allow it to be run on your computer. Note that using Viper like this will NOT automatically update it. Double clicking the file after doing this will launch Viper.
```

Comment:
It might be best to just link directly to the AppImage, [https://0negal.github.io/viper/?appimage](https://0negal.github.io/viper/?appimage) will always redirect to the latest `.AppImage`